flambe/experiment/experiment.py

```diff
@@ -274,8 +274,12 @@ def run(self, force: bool = False, verbose: bool = False, debug: bool = False, *
 
         # By default use all CPUs if no GPU is present
         devices = self.devices if self.devices else None
-        if devices is None and utils.local_has_gpu():
```

Comment:
I believe this is the only usage of this util so it should be deleted right?

Reply:
Yeah I think that's true, but I figured we might want to reuse that some day. Either way, it's gone in 0.5